README.md

```diff
@@ -48,42 +48,68 @@ Thinking about contributing to RES? Awesome! We just ask that you follow a few s
 
 ##### Top level files and folders
 
-  - `README.md`: YOU ARE HERE, unless you're browsing on GitHub
-  - `changelog.txt`: self-explanatory
-  - `gulpfile.babel.js`: build script
-  - `package.json`: package info, dependencies
+  - `assets/`: RES logo source
+  - `chrome/`: Chrome-specific RES files
+  - `dist/`: build output
+  - `edge/`: Microsoft Edge-specific RES files
+  - `examples/`: example code for new hosts/modules
+  - `firefox/`: Firefox-specific RES files
   - `lib/`: all RES code
   - `lib/core/`: core RES code
+  - `lib/css/`: RES css
+  - `lib/environment/`: RES environment code
+  - `lib/images/`: RES images
   - `lib/modules/`: RES modules
-  - `lib/vendor/`: RES vendor libraries
-  - `chrome/`: Chrome-specific RES files
-  - `firefox/`: Firefox-specific RES files
+  - `lib/templates/`: RES templates
+  - `lib/utils/`: RES utilities
+  - `lib/utils/__tests__/`: unit tests
+  - `lib/vendor/`: RES vendor libraries 
+  - `node/`: Node files
   - `safari/`: Safari-specific RES files
-  - `dist/`: build output
-  - `**/__tests__`: unit tests
+  - `utils/`: Misc RES utilities
+  - `CHANGELOG.md`: self-explanatory 
+  - `README.md`: YOU ARE HERE, unless you're browsing on GitHub
+  - `gulpfile.babel.js`: post-build script for zipping folders.
+  - `package.json`: package info, dependencies
+  - `webpack.config.babel.js`: build script
 
 ##### Chrome files
 
-  - `background.js`: the "background page" for RES, necessary for Chrome extensions
-  - `manifest.json`: the project manifest
+  - `images/css-off-small.png`, `images/css-off.png`, `images/css-on-small.png`, `images/css-on.png`: icons for css disable button.
+  - `background.entry.js`: the "background page" for RES, necessary for Chrome extensions
+  - `environment.js`: specific environment settings for Chrome
   - `icon.png`, `icon48.png`, `icon128.png`: icons!
+  - `manifest.json`: the project manifest
+  - `options.html`: options page for chrome extensions
 
+##### Microsoft Edge files
+
+  - `background-edge.html`: the "background page" for RES, necessary for Microsoft Edge extensions
+  - `edge.entry.js`: shim to allow chrome extension code usage
+  - `environment.js`: allows interceptor for Microsoft Edge permissions
```

Comment:
This is a bit too specific to what it does right now (we could have stuff other than interceptors), maybe something along the lines of `Edge-specific overrides of the Chrome environment`